gui/public/i18n/en/translation.ftl

```diff
@@ -282,10 +281,20 @@ settings-general-fk_settings-leg_tweak-floor_clip = Floor clip
 # why the name - without this enabled the feet will often slide across the ground as if your skating across the ground,
 # since this largely prevents this it corrects for it hence skating correction (note this may be renamed to sliding correction)
 # definition - Guesses when each foot is in contact with the ground and uses that information to improve tracking
+
+
+
```

Comment:
Why skip those lines?

Reply:
making room for myself and forgetting to remove it...